lightning/backend/importer_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	. "github.com/pingcap/check"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/kvproto/pkg/import_kvpb"
-	kvenc "github.com/pingcap/tidb/util/kvencoder"
+	kvenc "github.com/pingcap/kvproto/pkg/kvrpcpb"
```

Comment:
Since we only use the struct as a KV pair and nothing else, please we define our own `KvPair` struct containing only the `Key` and `Val` members. `kvrpcpb.KvPair` has too many irrelevant fields.